src/Bicep.Core.IntegrationTests/ScenarioTests.cs

```diff
@@ -33,6 +34,7 @@ param l
                 ("BCP028", DiagnosticLevel.Error, "Identifier \"l\" is declared multiple times. Remove or rename the duplicates."),
                 ("BCP079", DiagnosticLevel.Error, "This expression is referencing its own declaration, which is not allowed."),
                 ("BCP028", DiagnosticLevel.Error, "Identifier \"l\" is declared multiple times. Remove or rename the duplicates."),
+                (ParametersMustBeUsedRule.Code, DiagnosticLevel.Warning, new ParametersMustBeUsedRule().GetMessage()),
```

Comment:
>ParametersMustBeUsedRule [](http://example.com/codeflow?start=17&length=24)

nit - we should assert on a hard-coded string without a reference to the value because that will catch accidental renames, which would be breaking changes (once we have suppressions)